client/src/components/AlumniProfile.js

```diff
@@ -205,17 +260,46 @@ export default class AlumniProfile extends Component {
             />
             </>
         )
+
+        const interestsUpdate = (
+            <>
+            <Button
+                style={{'margin': '2px'}}
+                floated='right'
+                basic
+                color="blue"
+                onClick={this.openInterestsModal}
+            >
+                Add Interests
+            </Button>
+            <InterestsUpdateModal
+                modalOpen={this.state.interestsModalOpen}
+                closeModal={this.closeInterestsModal}
+                id={details._id}
+            />
+            </>
+        )
         var canUpdate;
 
         if (!isViewOnly) {
             canUpdate = (
+                <>
                 <Card.Content extra>
-                    {linkedInUpdate}
-                    {zoomLinkUpdate}
-                    {timeAvailabilitiesUpdate}
-                    {topicAvailabilitiesUpdate}
-                    {imageUpdate}
+                    <Grid centered columns={2}>
+                        <Grid.Column width={4}>
+                            {linkedInUpdate}
+                            {imageUpdate}
```

Comment:
Optional formatting: make this a vertical button group to make their widths identical